firebase-perf/src/main/java/com/google/firebase/perf/metrics/AppStartTrace.java

```diff
@@ -163,9 +166,16 @@ static AppStartTrace getInstance(TransportManager transportManager, Clock clock)
     this.clock = clock;
     this.configResolver = configResolver;
     this.executorService = executorService;
+
+    StartupTime startupTime = FirebaseApp.getInstance().get(StartupTime.class);
+    if (startupTime == null) {
+      firebaseStartupTime = new Timer();
+    }
+    firebaseStartupTime =
+        Timer.ofElapsedRealtime(TimeUnit.NANOSECONDS.toMillis(startupTime.getStartupNanos()));
```

Comment:
`SystemClock.elapsedRealtime()` and `System.nanoTime()` aren't in the same timebase, so assigning one to the other would not work (iirc nanoTime is when VM starts, elapsedRealtime is the last time the device booted). For context on why Fireperf chose `elapsedRealtime` see the other earlier comment.